ui/app/AppLayouts/Wallet/stores/WalletAssetsStore.qml

```diff
@@ -1,18 +1,25 @@
 import QtQuick 2.15
 
-import SortFilterProxyModel 0.2
 import StatusQ 0.1
+import StatusQ.Models 0.1
 import StatusQ.Core.Utils 0.1 as SQUtils
 
 import shared.stores 1.0
 
 import utils 1.0
 
+import SortFilterProxyModel 0.2
+
 QtObject {
     id: root
 
     property TokensStore walletTokensStore
 
+    // custom controller used for sorting/filtering (w/o source model)
+    readonly property var manageAssetsController: ManageTokensController {
```

Comment:
@caybro this seems to be not used anywhere?

Reply:
intended usage is the `SendModal` for example